libs/api/domains/official-journal-of-iceland-application/src/lib/ojoiApplication.service.ts

```diff
@@ -0,0 +1,29 @@
+import { OfficialJournalOfIcelandApplicationClientService } from '@island.is/clients/official-journal-of-iceland/application'
+import { Injectable } from '@nestjs/common'
+import { PostCommentInput } from '../models/postComment.input'
+import { PostApplicationInput } from '../models/postApplication.input'
+import { GetCommentsInput } from '../models/getComments.input'
+
+@Injectable()
+export class OfficialJournalOfIcelandApplicationService {
+  constructor(
+    private readonly ojoiApplicationService: OfficialJournalOfIcelandApplicationClientService,
+  ) {}
+
+  async getComments(input: GetCommentsInput) {
+    return await this.ojoiApplicationService.getComments(input)
```

Comment:
~ `return await` is redundant and only required when in a try/catch
```suggestion
    return this.ojoiApplicationService.getComments(input)
```